src/components/spinner/styled.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import { Colour, theme } from '@fellesdatakatalog/theme';
+import SpinnerImage from '../../images/spinner-digdir-fdk.svg';
+
+const Spinner = styled(SpinnerImage)`
+  & > svg {
+    width: 120px;
+    height: 120px;
+    border: 16px solid ${theme.colour(Colour.NEUTRAL, 'N50')};
+    border-top: 16px solid ${theme.colour(Colour.CYAN, 'C50')};
+    border-radius: 50%;
+  }
+`;
+
+export default { Spinner, SpinnerImage };
```

Comment:
Her blir selve `Spinner` en SVG, siden du legger stil direkte på `SpinnerImage` ved å pakke det inn med `styled()`.  Tror derfor ikke `& > svg` vil treffe den faktiske SVG'en!
```suggestion
const Spinner = styled(SpinnerImage)`
    width: 120px;
    height: 120px;
    border: 16px solid ${theme.colour(Colour.NEUTRAL, 'N50')};
    border-top: 16px solid ${theme.colour(Colour.CYAN, 'C50')};
    border-radius: 50%;
`;

export default { Spinner };
```

Om du ønsker å ha SVG'en inni et annet element kan du beholde koden som den er, men da for eksempel ha `const Spinner = styled.div` og beholde `index.tsx` som den er nå.